### DISCUSSION TITLE
NC_DB [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2095] [Discord Threads]
> 🙋‍♂️ **venom**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### You would have to use NC_DB / DATABASE_URL to a database to persist metadata   
####    
#### If this is not provided we default to sqlite and that gets lost when you reboot  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/getting-started/installation#production-setup  
#  
> 🙋‍♂️ **venom**  
#### got you, thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974372124611465246)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
